### DISCUSSION TITLE
问题已解决

### DISCUSSION THREAD

**Discussion by zhi1151013418** [getlantern/lantern-forum-cn#544] [Q&A]
期望可以改进下用户体验，祝好！

**Comment by MaxMKMax** [comment]
@zhi1151013418 您好. 请您也提供我支付单据. 
但这之前, 需要您帮我提交各应用内报告. 我们想了解为什么用不了翻墙 . 然后给我您的 8 位数票号. 谢谢您的帮忙.

**Reply by zhi1151013418** [reply]
1

**Reply by MaxMKMax** [reply]
@zhi1151013418 好的. 谢谢!
